src/Components/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  date: Date,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  clockId = 0;
+
+  componentDidMount() {
+    this.clockId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+
+      window.console.info(
+        this.state.date.toUTCString().slice(-12, -4),
+      );
```

Comment:
Don't use state here, because it can be outdated, save result of `new Date()` to constant and use it here